### PR TITLE
cloud: top navigation pr

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: top navigation pr

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/manage-user-access.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ If you are the organization owner, you can modify the system display time accord
 
 To change the local timezone setting, perform the following steps:
 
-1. Click the account name in the upper-right corner of the TiDB Cloud console.
+1. Click the buildng icon in the upper-right corner of the TiDB Cloud console.
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tidb-cloud/manage-user-access.md:213]: Is there an icon for it?

[the remark this answers is not in the document]
Comment (TomShawn) [reply tidb-cloud/manage-user-access.md:213]:
```suggestion
1. Click the <MDSvgIcon name="icon-top-organization" /> icon in the upper-right corner of the TiDB Cloud console.
```

File: tidb-cloud/create-tidb-cluster.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ For more information about the two options, see [Select Your Cluster Tier](/tidb
 
 If you are an organization owner, once you log in to TiDB Cloud, you have a default project. If you do not want to use the default project, you can take the following steps to create a new one:
 
-1. Click the account name in the upper-right corner of the TiDB Cloud console.
+1. Click the <MDSvgIcon name="icon-top-organization" /> icon in the upper-right corner of the TiDB Cloud console.
```
Comment (ran-huang) [inline tidb-cloud/create-tidb-cluster.md:39]: How about this?
```suggestion
1. Click <MDSvgIcon name="icon-top-organization" />**Organization** in the upper-right corner of the TiDB Cloud console.
```

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/tidb-cloud-support.md [hunk 3]
```diff
@@ -22,15 +22,15 @@ To request support for your organization, project, or cluster, perform the follo
 
 3. Click **Send**.
 
-Alternatively, you can click **Support** in the upper-right corner and click **Create New Case** to request support. If you go with this way, you need to provide your Organization ID in the request form as well. To get the Organization ID, click your account name in the upper-right corner of the TiDB Cloud console and click **Organization Settings**.
+Alternatively, you can click <MDSvgIcon name="icon-top-apply-for-poc" /> **Apply for PoC** icon in the upper-right corner and click **Create New Case** to request support. If you go with this way, you need to provide your Organization ID in the request form as well. To get the Organization ID, click <MDSvgIcon name="icon-top-organization" /> **Organization** in the upper-right corner of the TiDB Cloud console and click **Organization Settings**.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-support.md:25]:
```suggestion
Alternatively, you can click <MDSvgIcon name="icon-top-apply-for-poc" /> **Apply for PoC** in the upper-right corner and click **Create New Case** to request support. If you go with this way, you need to provide your Organization ID in the request form as well. To get the Organization ID, click <MDSvgIcon name="icon-top-organization" /> **Organization** in the upper-right corner of the TiDB Cloud console and click **Organization Settings**.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**